tests/functional/test_contribute.py

```diff
@@ -0,0 +1,51 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+from selenium.common.exceptions import TimeoutException
+
+from ..pages.contribute import ContributePage
+
+
+@pytest.mark.nondestructive
+def test_open_close_modal(base_url, selenium):
```

Comment:
I'd either rename this to `test_open_close_video_model` or perhaps simply `test_play_video`.